app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_promotion-block.less

```diff
@@ -0,0 +1,19 @@
+.promotion-block {
```

Comment:
the same here about the `& when (@media-common = true) {}`

Reply:
done